section-09-differential-testing/model-package/differential_tests/test_model_accuracy.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import numpy as np
+from regression_model.predict import make_prediction
+
+
+def test_make_prediction(sample_input_data, persist_results):
+    # Given
+    expected_first_prediction_value = 113422
+    expected_no_predictions = 1449
+
+    # When
+    result = make_prediction(input_data=sample_input_data)
+
+    # Then
+    predictions = result.get("predictions")
+    assert isinstance(predictions, list)
+    assert isinstance(predictions[0], np.float64)
+    assert result.get("errors") is None
+    assert len(predictions) == expected_no_predictions
+
+    # Persist for comparison
```

Comment:
do I understand correctly that this file is aimed to store the predictions so we can compare their values in a later script?

instead of doing this, which I find over-complicated, can we not just copy the notebook to this folder, re-run it with one extra cell where we store the prediction? it also serves to remind students of what we did, why we did it, and how we are going to use it now.